README.md

```diff
@@ -53,7 +37,8 @@ $ bin/spark-shell --packages com.springml:spark-salesforce_2.11:1.1.0
 * `dateFormat`: (Optional) A string that indicates the format that follow java.text.SimpleDateFormat to use when reading timestamps. This applies to TimestampType. By default, it is null which means trying to parse timestamp by java.sql.Timestamp.valueOf()
 * `resultVariable`: (Optional) result variable used in SAQL query. To paginate SAQL queries this package will add the required offset and limit. For example, in this SAQL query `q = load \"<dataset_id>/<dataset_version_id>\"; q = foreach q generate  'Name' as 'Name',  'Email' as 'Email';` **q** is the result variable
 * `pageSize`: (Optional) Page size for each query to be executed against Salesforce Wave. Default value is 2000. This option can only be used if `resultVariable` is set
-
+* `upsert`: (Optional) Indicator to either insert or update an existing record in one call. Default "false".
```

Comment:
* "indicator to either insert or update an existing record in one call" doesn't have a clear mapping to "true"/"false".
* Should document the behavior what happens to the existing fields that are not in the dataframe being pushed.
* Should mention `externalIdFieldName` here.
* What is the expected behavior when `upsert=true` used together with mode("append"), .e.g? If they shouldn't show up together. Maybe the best place to put `upsert` is under `mode`.

Reply:
Removed the `upsert` option. If mode is set to `SaveMode.Overwrite`, it will be an upsert.